poetry/core/masonry/builders/wheel.py

```diff
@@ -179,10 +181,23 @@ def _write_metadata(self, wheel):
     def _write_record(self, wheel):
         # Write a record of the files in the wheel
         with self._write_to_zip(wheel, self.dist_info + "/RECORD") as f:
+            record = StringIO()
+            if PY2:
+                record = BytesIO()
```

Comment:
```suggestion
            record = BytesIO() if PY2 else StringIO()
```
Minor suggestion to be a bit more concise here.

Reply:
Ok, as you like :) But I changed the order, to make clearer what's the default and we can just kick the end of the line once python2 isn't supported anymore